### PR TITLE
AF dir schema, based on examples

### PR DESCRIPTION
**Pull request by mccalluc**
The patterns could be tighter or looser... Ideally, the patterns specified here would be strict enough to satisfy the downstream pipelines, and capture any semantics intended by the producer.

- @pdblood : Can you review this, or identify someone who can?
- @cebriggs7135 : Can you supply descriptions, or identify someone who can?

### CODE REVIEW

**Review by cebriggs7135 — COMMENTED** [review]
I'm not sure that processedMicroscopy/transformations/transform.txt would be text files. If you know that for certain, then this looks OK to me.

**Comment by mccalluc** [comment]
@cebriggs7135 
> If you know that for certain

No, I don't: This is what the example had. I'm just making a tool: I don't think I should be the one to prescribe the structure.

File: src/ingest_validation_tools/directory-schemas/af.yaml [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+  description: 'TODO'
+-
+  pattern: 'processedMicroscopy/transformations/transform_\d+_[^/]+.txt'
+  description: 'TODO: "rigid", "affine", "custom"...'
+-
+  pattern: 'rawMicroscopy/[^/]\.xml'
+  description: 'TODO'
+-
+  pattern: 'rawMicroscopy/[^/]\.czi'
   description: 'TODO'
```
Comment (ropelews) [inline src/ingest_validation_tools/directory-schemas/af.yaml:15]:
```suggestion
# - A dataset consists of a single autofluorescence microscopy run, and  
#   a set of transformation files that are useful for other
#   related datasets. (Vanderbilt)

-
  pattern: 'processedMicroscopy/*_preMxIF_images/*.ome.tif'
  description: 'OME TIFF files (multichannel, multi-layered, image pyramids) produced by the autofluorescence microscopy linked to MxIF experiment'
-
  pattern: 'processedMicroscopy/*_preMxIF_transformations/*.txt'
  description: 'transformations to MxIF (related) data'
-
  pattern: 'processedMicroscopy/*_preIMS_images/*.ome.tif'
  description: 'OME TIFF files (multichannel, multi-layered, image pyramids) produced by the autofluorescence microscopy linked to IMS experiment'
-
  pattern: 'processedMicroscopy/*_preIMS_transformations/*.txt'
  description: 'transformations to IMS (related) data'
-
  pattern: 'processedMicroscopy/*_preCODEX_images/*.ome.tif'
  description: 'OME TIFF files (multichannel, multi-layered, image pyramids) produced by the autofluorescence microscopy linked to CODEX experiment'
-
  pattern: 'processedMicroscopy/*_preCODEX_transformations/*.txt'
  description: 'transformations to CODEX (related) data'
-
  pattern: 'rawMicroscopy/*.xml'
  description: 'XML metadata file from the autofluorescence microscopy experiments'
-
# Note: extension might change depending on the microscope used. This is just for Zeiss
# which is the only type that we are currently receiving.
  pattern: 'rawMicroscopy/*.czi'
  description: 'raw microscope file for the experiment'

```

**Comment by mccalluc** [comment]
Org builds stalled... going ahead with merge.

**Pull request merged**